app/content/objects/doorStyles.ts

```diff
@@ -898,6 +1041,28 @@ export const doorStyles: {[key: string]: DoorStyleDefinition} = {
     future: futureDoorStyle,
     futureDownstairs: stairsDoorStyle(futureDoorStyle, futureStairsDown, 'down', 6),
     futureUpstairs: stairsDoorStyle(futureDoorStyle, futureStairsUp, 'up', 6),
+    vanara: vanaraDoorStyle,
+    vanaraDownstairs: stairsDoorStyle(vanaraDoorStyle, vanaraStairsDown, 'down'),
+    vanaraUpstairs: stairsDoorStyle(vanaraDoorStyle, vanaraStairsUp, 'up'),
+    jadeLight: jadeLightDoorStyle,
+    jadeLightDownstairs: stairsDoorStyle(jadeLightDoorStyle, jadeLightStairsDown, 'down'),
+    jadeLightUpstairs: stairsDoorStyle(jadeLightDoorStyle, jadeLightStairsUp, 'up'),
+    jadeDark: jadeDarkDoorStyle,
+    jadeDarkDownstairs: stairsDoorStyle(jadeDarkDoorStyle, jadeDarkStairsDown, 'down'),
+    jadeDarkUpstairs: stairsDoorStyle(jadeDarkDoorStyle, jadeDarkStairsUp, 'up'),
+    jadeLightInt1: jadeLight1DoorStyle,
```

Comment:
Let's rename the actual styles to `jadeLightInterior...` and `jadeDarkInterior...` rather than using `Int1` here.